analytics/Makefile

```diff
@@ -144,6 +145,20 @@ lint: ## runs code quality checks
 # Data Commands #
 #################
 
+init-db:
+	@echo "=> Initializing the database schema"
+	@echo "====================================================="
+	$(POETRY) analytics etl initialize_database
+	@echo "====================================================="
+
+gh-transform-and-load:
+	@echo "=> Transforming and loading GitHub data into the database"
+	@echo "====================================================="
+	$(POETRY) analytics etl transform_and_load \
+	--deliverable-file $(DELIVERY_FILE) \
+	--effective-date $(EFFECTIVE_DATE)
+	@echo "====================================================="
+
```

Comment:
I also added these because I wasn't able to trigger the command from the natively installed python application because I don't have the `pyscopg_c` binding installed in my computer (which is needed by psycopg, but doesn't get distributed directly with the python library)

Reply:
Nice add. I assumed this would be needed in the near future, but had not spent any time on it yet. Thanks for adding it!